armi/utils/asciimaps.py

```diff
@@ -69,6 +70,8 @@ def readMap(self, text):
             for col, char in enumerate(line.split()):
                 i, j = self._getIndices(base, col)
                 self.lattice[i, j] = char
+
+        self.lattice = {k: v for k, v in self.lattice.items() if v != "-"}
```

Comment:
we should probably constantize this with 

`PLACEHOLDER = "-"` and then refer to `PLACEHOLDER` here.